packages/patternfly-4/react-core/src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const AccordionItem = ({ children, className, 'aria-labelledby': ariaLabelledBy, ...props }) => (
```

Comment:
It looks like className and aria-labelledby aren't being applied to the AccordionItem

Reply:
👍 just updated